### PR TITLE
Reduce number of Chromatic snapshots for Tooltip and Popover components

### PR DESCRIPTION
**Pull request by beaesguerra**
## Summary:

In https://github.com/Khan/wonder-blocks/pull/3072 and https://github.com/Khan/wonder-blocks/pull/3070, StateSheets were added for Popover and Tooltip components. 

In this PR, we disable more stories that are covered in those statesheets. 

Note: For popperjs floating components, they will only show up in the Chromatic snapshot if it is included in the initial viewport. Because of this, I set up a PopoverContent StateSheet too for more coverage of different ways it can be used. The Popover StateSheet is used to verify the appearance of an anchor + popover content

Issue: WB-2166

## Test plan:

Review snapshots in Chromatic and confirm we have enough coverage for Popover and Tooltip components

In the [Popover PR Chromatic build](https://www.chromatic.com/build?appId=5e1bf4b385e3fb0020b7073c&number=9243), there were 521 visual tests. [This PR Chromatic build](https://www.chromatic.com/build?appId=5e1bf4b385e3fb0020b7073c&number=9245) now has 509 tests! 

| Before | After | 
| --- | --- |
| Screenshot 2026-05-26 at 10 44 43 AM | Screenshot 2026-05-26 at 10 44 22 AM |

### CODE REVIEW

File: __docs__/wonder-blocks-popover/popover-content-testing-snapshots.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        chromatic: {
+            modes: allThemeModes,
+        },
+    },
+    tags: ["!autodocs", "!manifest"],
+} satisfies Meta<typeof PopoverContent>;
+
+export default meta;
+
+const rows = [
```
Comment (beaesguerra) [inline __docs__/wonder-blocks-popover/popover-content-testing-snapshots.stories.tsx:27]: These cover the examples in popper-content.stories.tsx so those stories can be disabled! 

PopoverContent is not rendered as a floating element so it's okay for this story to be long since all the examples will be included in the snapshot

File: __docs__/wonder-blocks-popover/popover-testing-snapshots.stories.tsx [hunk 2]
```diff
@@ -26,6 +26,8 @@ const meta = {
 
 export default meta;
 
+// Note: Favour coverage in PopperContent StateSheet since the Popover StateSheet
+// snapshot will only show the opened Popovers that are in the initial viewport.
```
Comment (beaesguerra) [inline __docs__/wonder-blocks-popover/popover-testing-snapshots.stories.tsx:30]: Limiting the examples in Popover's StateSheet since we're limited to what we can show in the initial viewport and it's covered by the PopoverContent StateSheet instead

File: __docs__/wonder-blocks-popover/popover.stories.tsx [hunk 3]
```diff
@@ -50,6 +50,11 @@ export default {
                 version={packageConfig.version}
             />
         ),
+        chromatic: {
+            // Disabling most snapshots in favour of statesheet. Explicitly
+            // enabling snapshots for specific stories.
+            disableSnapshot: true,
```
Comment (beaesguerra) [inline __docs__/wonder-blocks-popover/popover.stories.tsx:56]: Inverting the logic so all stories are disabled by default, and intentionally enable snapshots for specific stories instead

File: __docs__/wonder-blocks-popover/popover.stories.tsx [hunk 4]
```diff
@@ -774,6 +720,12 @@ export const PopoverAlignment: StoryComponentType = {
             <BasePopoverExample placement="left" />
         </View>
     ),
+    parameters: {
+        chromatic: {
+            // Include snapshot for alignment examples
+            disableSnapshot: false,
```
Comment (beaesguerra) [inline __docs__/wonder-blocks-popover/popover.stories.tsx:726]: It's helpful to have a snapshot for this alignment example to verify it continues to look okay. This will only take a snapshot for the default theme!

I think we don't need coverage for this across all themes since the alignment should be theme agnostic!

File: __docs__/wonder-blocks-popover/popover.stories.tsx [hunk 5]
```diff
@@ -980,6 +908,10 @@ export const InCorners: StoryComponentType = {
     },
     parameters: {
         layout: "fullscreen",
+        chromatic: {
+            // Include snapshot for corner alignment examples
+            disableSnapshot: false,
```
Comment (beaesguerra) [inline __docs__/wonder-blocks-popover/popover.stories.tsx:913]: Another alignment snapshot

File: __docs__/wonder-blocks-tooltip/tooltip.stories.tsx [hunk 6]
```diff
@@ -79,8 +79,12 @@ export default {
                 version={packageConfig.version}
             />
         ),
-        // Added to ensure that the tooltip is rendered using PopperJS.
-        chromatic: {delay: 500},
+        chromatic: {
+            // Added to ensure that the tooltip is rendered using PopperJS.
+            delay: 500,
+            // Visual coverage is provided by the Tooltip StateSheet snapshot.
+            disableSnapshot: true,
```
Comment (beaesguerra) [inline __docs__/wonder-blocks-tooltip/tooltip.stories.tsx:86]: Disable all snapshots by default and explicitly opt into snapshots based on the story

File: __docs__/wonder-blocks-tooltip/tooltip.stories.tsx [hunk 7]
```diff
@@ -561,6 +554,7 @@ export const InCorners = {
                 <Tooltip
                     {...args}
                     content="This is an example descriptor that's long with more content to see if it will display properly in different browsers"
+                    opened={true}
```
Comment (beaesguerra) [inline __docs__/wonder-blocks-tooltip/tooltip.stories.tsx:557]: Opening all the tooltips by default so we can have visual regression tests for the corner cases!

File: __docs__/wonder-blocks-popover/popover.stories.tsx (hunk 4 above)
Comment (jandrade) [reply __docs__/wonder-blocks-popover/popover.stories.tsx:726]: Good point, that makes sense :)

**Review by jandrade — APPROVED** [review]
Changes look great! thanks for optimizing these snapshots 👏
